app/adapters/application.js

```diff
@@ -1,16 +1,9 @@
-// import Ember from 'ember';
-// import FirebaseAdapter from 'emberfire/adapters/firebase';
+import DS from 'ember-data';
 
-// const { inject } = Ember;
+export default DS.JSONAPIAdapter.extend(DS.BuildURLMixin, {
```

Comment:
with the new version of `ember-writer`, you only need
`export { default } from 'ember-writer/adapters/application';`